docs/extensions/services/accessing-database.md

```diff
@@ -0,0 +1,72 @@
+---
+description: Learn about the ItemsService in Directus and how to utilize them when building extensions.
+contributors: Esther Agbaje
+---
+
+# Accessing Items in the Database
+
+The `ItemsService` allows you to work with and retrieve items data by creating endpoints to items in a collection. It
+provides a way to perform CRUD operations on items in a database.
+
+## Create an Item in a Specific Collection
```

Comment:
```suggestion
## Create an Item
```

I would drop 'in a specific collection' across the board because they are all in a specific collection. Directus users should understand the fact items must be inserted in a collection